posting/index.go

```diff
@@ -561,7 +561,6 @@ func (r *rebuilder) Run(ctx context.Context) error {
 		WithNumVersionsToKeep(math.MaxInt32).
 		WithLogger(&x.ToGlog{}).
 		WithCompression(options.None).
-		WithEncryptionKey(x.WorkerConfig.EncryptionKey).
```

Comment:
Why do we need to remove this?

Reply:
We are already setting the encryption key a few lines below. This was redundant.